sdk/testing/src/main/java/io/opentelemetry/sdk/testing/logs/TestLogRecordData.java

```diff
@@ -24,6 +25,9 @@
  */
 @Immutable
 @AutoValue
+@AutoValue.CopyAnnotations
+@SuppressWarnings("deprecation") // Carry suppression for Body to AutoValue implementation via
+// @AutoValue.CopyAnnotations
```

Comment:
this line looks like commented out code due to the wrapping placement, maybe do a bit of manual wrapping?